File: ventasBackend/models/venta.js
/* const sequelize = require("../database/database");
const { DataTypes } = require('sequelize');
const Sucursal = require('./sucursal');

const Venta = sequelize.define('ventas', {
    id: {
        type: DataTypes.INTEGER,
        primaryKey: true,
        autoIncrement: true
    },
    fecha:{
        type: DataTypes.DATE, 
        defaultValue: DataTypes.NOW       
    },
    total:{
        type: DataTypes.FLOAT,
        defaultValue: 0
    },
    ganancias:{
        type: DataTypes.FLOAT,        
        defaultValue: 0
    },
    descuento:{
        type: DataTypes.FLOAT,
        defaultValue: 0
    },
    estado: {
        type: DataTypes.BOOLEAN,
        defaultValue: true,
    },
    idsucursales: {
        type: DataTypes.INTEGER,
        references: {
            model: Sucursal, 
            key: 'id'
        }
    },
}, {
    timestamps: false, 
});

module.exports = Venta;

 */

File: ventas_app/lib/features/ventas/domain/entities/venta.dart
class Venta {
  int id;
  DateTime fecha;
  double total;
  double ganancias;
  double descuento;
  int idsucursales;

  Venta(
      {required this.id,
      required this.fecha,
      required this.total,
      required this.ganancias,
      required this.descuento,
      required this.idsucursales});
}
